noise-maker/src/components/key.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Key = ({keyClick, label}) => {
+  return(
+    <div onClick={keyClick} className="key">
```

Comment:
Clarification question. This is like a helper function right? Is it a functional component (vs class)?

Reply:
is just a class that is used to style the keys